addons/account/models/partner.py

```diff
@@ -354,6 +354,11 @@ def _credit_debit_get(self):
         remaining.debit = False
         remaining.credit = False
 
+    @api.depends_context('company')
+    def _compute_credit_to_invoice(self):
+        # To be overridden in Sales
+        self.credit_to_invoice = False
```

Comment:
```suggestion
        pass
```

Reply:
I have to assign a value to the field in the compute method, otherwise I get:
```
ValueError: Compute method failed to assign res.partner(249,).credit_to_invoice
```

Reply:
Right, the field is not stored, my bad :wink: